its/ruling/src/test/resources/guava/squid-S2388.json

```diff
@@ -4,6 +4,11 @@
 1618,
 1665,
 ],
+'com.google.guava:guava:src/com/google/common/collect/AbstractBiMap.java':[
+424,
```

Comment:
These are FPs to me, these calls are legit.

Reply:
These calls are calling super class methods from an inner class which makes them relevant to be highlighted by the rule.